tool.go

```diff
@@ -376,17 +378,39 @@ func main() {
 					return err
 				}
 
-				tempfile, err := ioutil.TempFile(currentDirectory, "test.")
-				if err != nil {
-					return err
+				if *compileOnly && len(*outputFilename) == 0 {
+					*outputFilename = pkg.Package.Name + "_test.js"
+				}
+
+				var outfile *os.File
+				if len(*outputFilename) > 0 {
+					err := os.Remove(*outputFilename)
```

Comment:
Are those `Remove` calls necessary? Shouldn't `Create` just overwrite the files?